src/types.ts

```diff
@@ -26,27 +27,32 @@ export type ValidInvocationPayloadBody =
 // Invocation Bodies
 export type FunctionInvocationBody = {
   event: {
-    type: "function_executed";
+    type: typeof EventTypes.FUNCTION_EXECUTED;
     function: {
       callback_id: string;
     };
     function_execution_id: string;
     inputs: FunctionInputValues;
-    bot_access_token?: string;
+    bot_access_token: string;
```

Comment:
In my testing in both enterprise and regular workspaces, this property was always present.